CRM/Upgrade/Incremental/php/SixOne.php

```diff
@@ -29,6 +29,36 @@ class CRM_Upgrade_Incremental_php_SixOne extends CRM_Upgrade_Incremental_Base {
    */
   public function upgrade_6_1_alpha1($rev): void {
     $this->addTask(ts('Upgrade DB to %1: SQL', [1 => $rev]), 'runSql', $rev);
+    $this->addTask('Increase field length of civicrm_action_schedule.entity_status', 'alterSchemaField', 'ActionSchedule', 'entity_status', [
+      'title' => ts('Entity Status'),
+      'sql_type' => 'varchar(255)',
+      'input_type' => 'Select',
+      'description' => ts('Entity status'),
+      'add' => '3.4',
+      'serialize' => CRM_Core_DAO::SERIALIZE_SEPARATOR_TRIMMED,
+      'input_attrs' => [
+        'label' => ts('Entity Status'),
+        'multiple' => '1',
+        'control_field' => 'entity_value',
+      ],
+      'pseudoconstant' => [
+        'callback' => ['CRM_Core_BAO_ActionSchedule', 'getEntityStatusOptions'],
+      ],
+    ]);
+    $this->addTask('Increase field length of civicrm_action_schedule.start_action_date', 'alterSchemaField', 'ActionSchedule', 'start_action_date', [
+      'title' => ts('Start Action Date'),
+      'sql_type' => 'varchar(255)',
+      'input_type' => 'Select',
+      'description' => ts('Entity date'),
```

Comment:
Not a big deal (it's just the upgrader), but why 'Entity date' instead of 'Start Action Date'?

Reply:
Not really sure. All these columns are multipurpose depending on the entity selected so maybe this was trying to be super generic.